cmd/signature-v4.go

```diff
@@ -417,3 +419,36 @@ func doesSignatureMatch(hashedPayload string, r *http.Request, validateRegion bo
 	// Return error none.
 	return ErrNone
 }
+
+// Returns presignedGET url.
+// Called From MinioBrowser via json-RPC.
+func presignedGet(host, bucket, object string) string {
+	cred := serverConfig.GetCredential()
+	region := serverConfig.GetRegion()
+
+	accessKey := cred.AccessKeyID
+	secretKey := cred.SecretAccessKey
+
+	date := time.Now().UTC()
+	dateStr := date.Format("20060102T150405Z")
+	credential := fmt.Sprintf("%s/%s", accessKey, getScope(date, region))
+
+	query := strings.Join([]string{
+		"X-Amz-Algorithm=" + signV4Algorithm,
+		"X-Amz-Credential=" + strings.Replace(credential, "/", "%2F", -1),
+		"X-Amz-Date=" + dateStr,
+		"X-Amz-Expires=" + "604800",
+		"X-Amz-SignedHeaders=host",
+	}, "&")
+
+	path := "/" + path.Join(bucket, object)
+
+	var extractedSignedHeaders http.Header // will be nil as "host" is the only signed header.
```

Comment:
I couldn't understand the comment about ``host``  being the only signed header. Could you explain that?